src/aws-credentials/provider.cr

```diff
@@ -74,11 +74,7 @@ module Aws::Credentials
     end
 
     def refresh : Nil
-      @providers.each do |provider_|
-        provider_.refresh
-      rescue
-        # Nothing to do
-      end
+      @providers.each(&.refresh) rescue nil
```

Comment:
why just not use method level rescue?

Reply:
It's a little shorter. The method is trivial anyway.